apps/mobile/src/urql-shared/updates/index.js

```diff
@@ -107,10 +107,12 @@ export default {
         }
         case 'MessageThread': {
           console.log('!!!! updates TODO: new MessageThread. Increment Messages tab badge', result, args)
+          cache.invalidate({ __typename: 'MessageThread', id: update?.messageThread?.id })
```

Comment:
Enables proper live updating to MessageThreads and Messages in he background, and working well now that the pagination caching issue is cleared-up.